crates/accelerate/src/basis/basis_translator/mod.rs

```diff
@@ -351,8 +345,8 @@ fn extract_basis_target(
                 unreachable!("Control flow op is not a control flow op. But control_flow is `true`")
             };
             let bound_inst = op.instruction.bind(py);
-            // Use python side extraction instead of the Rust method `op.blocks` due to
-            // required usage of a python-space method `QuantumCircuit.has_calibration_for`.
+            // TODO: Use Rust method `op.blocks` instead of Python side extraction now that
+            // the usage of a python-space method `QuantumCircuit.has_calibration_for` is not needed anymore
```

Comment:
It's a minor suggestion but I wasn't sure from the wording if the method had been removed or not:

```suggestion
            // TODO: Use Rust method `op.blocks` instead of Python side extraction now that
            // the python-space method `QuantumCircuit.has_calibration_for`
            // has been removed and we don't need to account for it.
```